docs/architecture/architecture-and-security.md

```diff
@@ -0,0 +1,253 @@
+To deploy the Central Landscaper you have to provide two kubeconfigs as Helm values, one with admin rights to access 
+the cluster where the Central Landscaper is running and one providing access to the cluster, where the Landscaper 
+resources (Installations, Deploy Items, Targets etc.) are maintained. For the Central Landscaper these both 
+kubeconfigs point to the same cluster Core-Shoot-Cluster. The Deploy-Pipeline must create and update these 
+kubeconfigs with every execution. The Deploy-Pipeline creates and rotates each of these kubeconfigs as follows:
+
+- With the Gardener-Service-Account-Kubeconfig for the Gardener-LaaS-Project, create a Shoot-Cluster-Admin-Kubeconfig 
+  for Core-Shoot-Cluster.
+
+- Use the Shoot-Cluster-Admin-Kubeconfig for the Core-Shoot-Cluster to create a service account with the required 
+  permissions for the Core-Shoot-Cluster and create a Shoot-Cluster-Kubeconfig for this service account.
+
+**Credentials when deploying the LaaS:** To deploy the LaaS the Deploy-Pipeline creates an Installation and a Target
+on the Core-Shoot-Cluster. This is executed with a Shoot-Cluster-Admin-Kubeconfig for the Core-Shoot-Cluster, fetched
+as described before. The Target is created and rotated by the Deploy-Pipeline as follows:
+
+- With the Gardener-Service-Account-Kubeconfig for the Gardener-LaaS-Project, create a Shoot-Cluster-Admin-Kubeconfig 
+  for Core-Shoot-Cluster.
+
+- Use the Shoot-Cluster-Admin-Kubeconfig for the Core-Shoot-Cluster to create a service account with the required
+  permissions for the Core-Shoot-Cluster and create a Shoot-Cluster-Kubeconfig for this service account. This kubeconfig
+  is stored in the Target (or the referenced secret).
+
+The Installation to deploy the LaaS requires further credentials, mainly to be able to 
+
+- create shoot k8s resources in the Gardener-Resource-Cluster-Project
+- provide the Landscaper instances on the Target-Shoot-Cluster access to their Resource-Shoot-Clusters
+
+For these credentials the Deploy-Pipeline creates a secret as input for the Installation, containing the 
+Gardener-Service-Account-Kubeconfig for the Gardener-Resource-Cluster-Project stored in the Secret Store.
+
+### List of Credentials for Rotation
+
+- Manually rotated credentials in Secret Store
+  - Gardener-Service-Account-Kubeconfig for Gardener-LaaS-Project
+  - Gardener-Service-Account-Kubeconfig for Gardener-Resource-Cluster-Project
+  - Secret key for the GCP service account of the two GCP projects referenced by the shoot clusters
+
+- Automatically rotated by the Deploy-Pipeline
+  - Secrets containing the GCP secret keys in the Garden-Projects Gardener-LaaS-Project and 
+    Gardener-Resource-Cluster-Project referenced by shoot resources 
+  - Credentials stored in the ServiceTargetConfigs
+  - 2 Shoot-Cluster-Kubeconfigs required as Helm values to install the Central Landscaper 
```

Comment:
Since both of them are pointing to the same cluster, only one is needed.

Reply:
You are right. This is anyhow something which should be described in more detail. The installations of the central landscaper consists of the installations of 2 helm charts. I think this is quite important.

Reply:
Described in more detail